doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1990,30 +1990,76 @@ Response:
 
 # Group Registrations
 
-Context Registration allows to associate external services to context data. One of the main
-use cases of this functionality is the association of Context Providers.
+A Context Registration allows to bind external context information sources so that they can be
+play the role of providers
+of certain subsets (entities, attributes) of the context information space, including those located
+at specific geographical areas.
+A compliant Context Information provider shall expose an NGSI10 (legacy) or an NGSIv2 interface, in particular:
```

Comment:
I would adopt the same approach of the legacy mode for notifications so I don't describe legacy forwarding in the NGSIv2 .apib. In this sense, I'd suggest to remove this sentence.

Reply:
fixed 78ce71dfa1b2c6952018ede3a9ef640c65abc1dd